stat/stat_test.go

```diff
@@ -1524,8 +1532,9 @@ func TestQuantile(t *testing.T) {
 				if test.panics {
 					continue
 				}
-				if v != test.ans[k][j] && !(math.IsNaN(v) && math.IsNaN(test.ans[k][j])) {
-					t.Errorf("mismatch case %d kind %d percentile %v. Expected: %v, found: %v", i, k, p, test.ans[k][j], v)
+				expected := test.ans[k][j]
```

Comment:
Just easier to handle floats imprecision  if we use this. e.g., `1.90000000001 != 1.9` failing

Reply:
We write `expected` as `want` (or in this file, `ans`).